css/style.css

```diff
@@ -0,0 +1,14 @@
+body{
+  text-align: center;
+  border: 3px solid green;
+}
+#navbar {
+  position: fixed;
+  height: 100%;
+  top: 0;
+  left: 0;
+}
+@media screen and (max-height: 450px) {
+  #navbar  {padding-top: 15px;}
+  #navbar  a {font-size: 18px;}
+}
```

Comment:
Please add more css styles to arrange all the parts of the website in their natural order.